libres/lib/include/ert/enkf/enkf_plot_gendata.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <ert/util/type_macros.h>
 #include <ert/util/double_vector.h>
+#include <ert/util/int_vector.h>
```

Comment:
Not used?

Reply:
Bad change manage on my part, that had already been cleaned up, just not pushed :-)